gridstatus/logging.py

```diff
@@ -0,0 +1,23 @@
+import logging
+import sys
+
+
+def configure_logging():
+    logger = logging.getLogger("gridstatus")
+    if not len(logger.handlers):
+        logger.setLevel(logging.INFO)
+        fmt = ("%(name)s: %(levelname)s: %(message)s",)
+        # by default, StreamHandler logs to stderr
+        handler = logging.StreamHandler(sys.stderr)
+        # log info, debug, warning, error, critical to stdout
+        handler.setLevel(logging.INFO)
+        handler.setFormatter(logging.Formatter(fmt=fmt))
+        logger.addHandler(handler)
+    return logger
+
+
+def log(msg, verbose=False):
+    if verbose is False:
+        return
+    logger = configure_logging()
```

Comment:
if you update this function to just use a print rather than python logging, im good to merge this is in.

in the future, if we want to use python logging this refactor will make it super easy to do

Reply:
Fixed:
https://github.com/kmax12/gridstatus/pull/153/commits/3d390236144a9e78e7619f3a7681352635044603